http-core/src/main/scala/org/apache/pekko/http/impl/engine/http2/Http2SettingsHeader.scala

```diff
@@ -42,4 +48,5 @@ private[pekko] object Http2SettingsHeader {
     val reader = new io.ByteStringParser.ByteReader(bytes)
     Http2FrameParsing.readSettings(reader, log)
   }
+
```

Comment:
remove this new line

Reply:
Does the removed method been used anywhere ,eg in grpc module?

Reply:
it's package private code so it would be really bad practice to use it outside of this module

Reply:
I've just checked incubator-pekko-grpc, incubator-pekko-connectors and incububator-pekko-management git repos and can't find any uses of the class that I am removing in this PR.

Reply:
I've committed a change that removes the new line